aspnet/mvc/models/validation/sample/Startup.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.Hosting;
+using Microsoft.AspNet.Identity.EntityFramework;
+using Microsoft.Data.Entity;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using MVCMovie.Models;
+using MVCMovie.Services;
+using Microsoft.AspNet.Mvc;
+
+namespace MVCMovie
+{
+    public class Startup
```

Comment:
Pare this class down significantly. Seems to contain a lot that isn't relevant e.g. logging, user secrets, identity, authorization, `IEmailSender`, ...